nameko/rpc.py

```diff
@@ -507,10 +504,12 @@ class Client(object):
     """
 
     def __init__(
-        self, publish, register_for_reply, service_name=None, method_name=None
+        self, publish, register_for_reply, context_data,
```

Comment:
Might want to add `context_data` to `:Parameters:` doc above.

Reply:
good catch, thanks! .)